deploy-kyma-keda-k3d.sh

```diff
@@ -0,0 +1,39 @@
+#! /bin/bash
+
+# Add those to /etc/hosts
+# 127.0.0.1 k3d-kyma-registry.localhost
+# 127.0.0.1 k3d-kyma-registry
+
+# Provision K3D cluster
+kyma provision k3d --ci
```

Comment:
Should we use the same instruction from the README? As in k3d directly not using the kyma cli?

Reply:
Yes. good point. I will update readme though, because `kyma provision k3d` allows you to install kyma on top of that (initialisies the registry exaclty how kyma deploy command expects it)